java/client/src/test/java/glide/api/RedisClusterClientTest.java

```diff
@@ -96,7 +97,8 @@ public TestCommandManager(Response responseToReturn) {
 
         @Override
         public <T> CompletableFuture<T> submitNewCommand(
-                Command command, RedisExceptionCheckedFunction<Response, T> responseHandler) {
+                RedisRequestOuterClass.RedisRequest.Builder command,
```

Comment:
```suggestion
                RedisRequest.Builder command,
```

Reply:
So strange.  I think my IDE is adding these when I run "Optimize imports"... need to check.  Because I didn't add any lines of code.

Reply:
IDEA dislikes importing classed defined in other classes.